### PR TITLE
Add TypeScript definition for the module

### PR DESCRIPTION
**Pull request by vesse**
Add TypeScript definition

```typescript
import * as virtuaaliviivakoodi from 'virtuaaliviivakoodi';

const opts: virtuaaliviivakoodi.Options = {
  iban: 'FI37 1590 3000 0007 76',
  reference: 11112,
  amount: 12.25,
  due: '161221'
};

console.log(virtuaaliviivakoodi(opts));
```

### CODE REVIEW

**Comment by vesse** [comment]
Needs work apparently, nevermind.

**Comment by vesse** [comment]
Sorry for being so hasty - was just importing it wrong.

**Comment by akselinurmio** [comment]
Thanks for the PR @vesse! Much appreciated. I’ll look into it as soon as I have a chance. :)

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+     */
+    iban: string;
+    /**
+     * Valid Finnish bank reference or ISO 11649 Creditor Reference
+     */
+    reference: number;
+    /**
+     * Amount in EUR
+     */
+    amount: number;
```
Comment (akselinurmio) [inline index.d.ts]: This would be `reference?: number`

**Comment by vesse** [comment]
I changed amount and due to optionals, and reference can of course be a string too in case of RF ref.

**Comment by akselinurmio** [comment]
>Reference is optional too, and can be string in case of RF ref
>Reference is not after all optional after looking at the implementation

Sorry, this was my mistake! Glad you figured it out. I run the prettier on your branch, so we should be ready for merge now.

Thanks a ton for you PR!

**Comment by vesse** [comment]
Thanks! Care to publish to npm too?

**Comment by akselinurmio** [comment]
Update is on its way. :)

**Pull request merged**